SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityScheduledJobPoller.java

```diff
@@ -112,10 +112,20 @@ public void runActionOnPoll() {
 
       final CronExpression cronExpression;
 
+      Optional<String> maybeSchedule = request.getRequest().getQuartzScheduleOrSchedule();
+      if (!maybeSchedule.isPresent()) {
```

Comment:
Yeah I think we need to approach this differently -- the `getQuartzScheduleSafe()` method isn't great because it falls back to the cron schedule if no quartz schedule is set, which doesn't make much sense because the two syntaxes are incompatible with each other. I think the original PR to fix the validator is good enough for now.